base_layer/p2p/src/proto/tari.p2p.message_type.rs

```diff
@@ -15,8 +15,9 @@ pub enum TariMessageType {
     BaseNodeResponse = 70,
     MempoolRequest = 71,
     MempoolResponse = 72,
-    /// -- DAN Messages --
     TransactionFinalized = 73,
+    /// -- DAN Messages --
+    TransactionCancelled = 74,
```

Comment:
```suggestion
    TransactionCancelled = 74,
     // -- DAN Messages --
```

Reply:
Clicked "Add single comment" instead of "Start a review" :P

Reply:
I keep trying to do that but that is the generated file